site/src/pages/tutorials/tutorial-blog-rest-step4.mdx

```diff
@@ -0,0 +1,132 @@
+---
+menuTitle: "Implement READ"
+order: 4
+type: step
+tags:
+  - rest
+  - request-converter
+level: basic
+---
+
+# Implementing READ operation
+
+In this step, we'll implement two methods at once. One is for retrieving a single post and another for multiple blog posts. We'll map both methods with HTTP GET.
+
+<TutorialSteps current={4} />
+
+By completing this step, you'll learn to map your service with the HTTP GET (<type://@Get>) method, use parameter injection (<type://@Param>), set default parameter value (<type://@Default>), and return a JSON object (<type://@ProducesJson>) as a response.
+
+## What you need
+
+You need to have these files prepared from previous steps.
+
+- [BlogService.java](/tutorials/tutorial-blog-rest-step3#0-create-a-service-file)
+- [BlogPost.java](/tutorials/tutorial-blog-rest-step2)
+
+## 1. Map HTTP method
+
+Let's start with [mapping the HTTP GET method](/docs/server-annotated-service#mapping-http-service-methods) with our service methods:
+
+1. [Create a service file](/tutorials/tutorial-blog-rest-step3#0-create-a-service-file) if you're yet to create one.
+2. Declare service methods, `getBlogPost()` and `getBlogPosts()` in the class `BlogService`.
+3. Map these service methods with the HTTP GET method by adding the <type://@Get> annotation as follows.
+4. Assign the endpoint as `/blogs`.
```

Comment:
nit:
Is `bind`  more appropriate for this context?
`Bind the endpoint to /blogs.` 🤔

Reply:
Changed the pattern to "Bind the endpoing `/abc` to the method. Hope this is okay.
bdcd5ad30585e3939c4effbc35616de7cc7501fe